timesync

```diff
@@ -12,33 +12,48 @@ import urllib.request
 if __name__ == '__main__':
     home = os.getenv('HOME')
 
+    # Load the configuration file
     with open(os.path.join(home, '.timesync'), 'r') as config:
         config = json.load(config)
 
+        # Project - required
         project = input('Project: ')
+        # Activities - required
         activities = [input('Activity: ')]
+        # More activities if necessary
         while(input('More activities? [y/n] ') == 'y'):
             activities.append(input('Activity: '))
 
+        # Duration in seconds - required
         duration = int(input('Time worked in seconds: '))
         issue_uri = input('Issue uri: ') or None
 
+        # Date Worked
         d = datetime.datetime.now()
+        # Month worked - optional
         month_worked = input('Month worked (format: 01): ')
```

Comment:
while you're here can you change these to be the ``or d.month`` thing I did with ``issue_uri`` above?